packages/app/src/containers/LayerList.tsx

```diff
@@ -207,9 +207,34 @@ export default memo(function LayerList() {
     selectedObjects.length === 1 &&
     items.find((i) => i.id === selectedObjects[0])?.type === 'symbolInstance';
 
+  const canBeSymbol = useCallback((): number => {
+    const selectedItems = items.filter((i) => selectedObjects.includes(i.id));
+
+    if (
+      selectedItems.length === 0 ||
+      selectedItems.some((l) => l.type === 'symbolMaster')
+    )
+      return 0;
+
+    if (
```

Comment:
It looks like the same condition for returning `1` and `2`? Are these supposed to be different? If not, we should return a boolean

Reply:
yes, they are supposed to be different, this could be handle by a type but I am not sure yet. It's because when you have artboard you don't ask for the symbol name 🤔

Reply:
We can probably make a separate boolean variable, like `shouldAskForSymbolName`, rather than re-using `canBeSymbol`. Ideally `canBeSymbol` will also be a boolean variable, rather than a function